sample/test/fails/test-fail-output-1.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main(int argc, char **argv)
+{
+	puts("[Test FAIL TIMEOUT started]");
+	puts("1");
+	puts("This line will not be parsed");
```

Comment:
add `while (1) 
;`

Program ending before the expected string should be detected as immediate failure instead of timeout

missing return?